cmd/notation/policy/import.go

```diff
@@ -24,11 +24,9 @@ func importCmd() *cobra.Command {
 		Short: "Import trust policy configuration from a JSON file",
 		Long: `Import trust policy configuration from a JSON file.
 
-** This command is in preview and under development. **
-
-Example - Import trust policy configuration from a file:
-  notation policy import my_policy.json
-`,
+** This command is in preview and under development. **`,
```

Comment:
This is incorrect, policy command is not under preview.
Can you please remove this statement? If you want to do it in next PR thats also fine as its an scope creep.